Gemfile

```diff
@@ -42,3 +42,6 @@ gem 'chronic'
 # for dot access to Hash members in HAML pages
 gem 'hash_dot', '~> 2.4.2'
 
+# for checking broken links
+gem 'rake'
```

Comment:
We do not run the tests using Rakefile, so it's unnecessary.

Reply:
ok